dawn/src/dawn/Optimizer/AccessComputation.cpp

```diff
@@ -317,8 +317,8 @@ class AccessMapper : public iir::ASTVisitor {
     removeLastChildAccesses();
   }
   virtual void visit(const std::shared_ptr<iir::ReductionOverNeighborExpr>& expr) override {
-    expr->getRhs()->accept(*this);
-    expr->getInit()->accept(*this);
+    for(auto& s : expr->getChildren())
```

Comment:
we usually also put `{}` around single statement for loops. Worth for consistency?

Reply:
Are you sure? Because if you look at this file, exactly these two lines appear many times (without `{}`). I think currently, consistency is not really enforced and I am tempted to use GCC coding convention (just use the code convention that is used around the code that you're modifying) ;-)

Reply:
:) Tendency is yes - Most likely it's not enforced but I think it was intended at some point. I can live with either

Reply:
I think, here I tend to skip the braces, because exactly this line is used many times in the same file. If we want to enforce the convention, we should do this in a separate PR and do all together.